modules/metrics_prometheus_v1/src/main/scala/com/evolutiongaming/skafka/metrics/KafkaMetricsCollector.scala

```diff
@@ -40,26 +40,25 @@ class KafkaMetricsCollector[F[_]: Monad: ToTry](
   customLabels: List[(String, String)],
 ) extends MultiCollector {
 
-  private val MetricNameRegex: Regex                 = "[a-zA-Z_:][a-zA-Z0-9_:]*".r
-  private val LabelNameRegex: Regex                  = "[a-zA-Z_][a-zA-Z0-9_]*".r
   private val (customLabelsKeys, customLabelsValues) = customLabels.separate
 
   override def collect(): MetricSnapshots = {
     for {
       metrics      <- kafkaClientMetrics
-      metricsGroups = metrics.groupBy(m => (m.name, m.group, m.description)).toList
```

Comment:
The same metrics but with different tags might have different description, such as 
The average number of bytes consumed per second
and
**The average number of bytes consumed per second for a topic**
Since the prometheus name of our final metric doesn't contain a description, we simply don't group by it and just take the first one in a group